src/crate/client/sqlalchemy/compat/api13.py

```diff
@@ -131,3 +131,26 @@ def monkeypatch_amend_select_sa14():
     sqlalchemy.select = select_sa14
     sqlalchemy.sql.select = select_sa14
     sqlalchemy.sql.expression.select = select_sa14
+
+
+@property
+def connectionfairy_driver_connection_sa14(self):
```

Comment:
Isn't this only added for SA < 1.4?
```suggestion
def connectionfairy_driver_connection_sa13(self):
```